### ISSUE TITLE
Markdown tasks list no longer working after upgrade to 1.2.1

### ISSUE THREAD

**Issue by broferek** [nextcloud/deck#2595]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
2 unexpected bahviours :
1. When using Markdown tasks list in a card, if you tick a particular task, all the description content suddenly disappears.
2. When ticking a task, the other tasks are unticked

**To Reproduce**
Steps to reproduce the first behavior:
1. Create a card with a task list (- [ ] foo\n-[ ] bar)
2. Save the description
3. Reload your page
4. Click on the card
5. Tick the `bar` task
6. All the content of the description is removed

Steps to reproduce the second behavior:
1. Create a card with a task list (- [ ] foo\n-[ ] bar)
2. Save the description
3. Tick the `bar` task
4. Tick the `foo` task
6. The `bar` task is unticked automatically

**Expected behavior**
The task is ticked and the content remains

**Client details:**
 - OS: Fedora 33
 - Browser : Chrome / Firefox
 - Version 86.0.4240.198 / 82.0.3
 - Device: desktop

<details>
<summary>Server details</summary>

**Operating system**: CentOS7

**Web server:** Apache

**Database:** Mysql

**PHP version:** 7.3.20

**Nextcloud version:** (see Nextcloud admin page) 19.0.1

**Where did you install Nextcloud from:** 

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by stefan-niedermann** [comment]
Duplicate of https://github.com/nextcloud/deck/issues/2580, https://github.com/nextcloud/deck/issues/2591 and #2592 

[A fix](https://github.com/nextcloud/deck/pull/2584) is already in the pipeline.

You should be able to [restore the description content from the timeline tab](https://github.com/nextcloud/deck/issues/2580#issuecomment-730319105).

Please use the search function next time before open a new issue, thank you 🙂.

**Comment by broferek** [comment]
I did search but using "markdown", "task", "list" keywords and just after my issue creation, I went in the closed issue and saw the good issues and that it was expected to be released in 1.2.2. Sorry for the duplicate.

**Comment by stefan-niedermann** [comment]
fyi @juliushaertl i pinned the original issue (#2580) regarding task lists, because this was the 4th report about this issue.
